src/ui/components/NewAccordion/utils.ts

```diff
@@ -0,0 +1,113 @@
+import { findLastIndex } from "lodash";
+import { Section } from "./reducer";
+
+export const MIN_HEIGHT = 150;
+export const ACCORDION_HEIGHT = 600;
+export const BORDER_HEIGHT = 1;
+export const HANDLE_HEIGHT = 4;
+export const HEADER_HEIGHT = 24 + BORDER_HEIGHT;
+
+// Utils
+const getSectionsTotalHeight = (sections: Section[]) => {
+  return sections.reduce((a, section) => {
+    return a + getActualHeight(section);
+  }, 0);
+};
```

Comment:
super nit but I would probably add a newline between these methods for legibility